interface/billing/billing_report.php

```diff
@@ -368,236 +387,293 @@ function MarkAsCleared(Type)
-$ThisPageSearchCriteriaDisplayRadioMaster[1] = array( xl("All"),xl("eClaims"),xl("Paper") );//Display Value
-$ThisPageSearchCriteriaRadioKeyMaster[1]="all,standard,hcfa";//Key
-$ThisPageSearchCriteriaDisplayRadioMaster[2]= array( xl("All"),xl("Insured"),xl("Non-Insured") );//Display Value
-$ThisPageSearchCriteriaRadioKeyMaster[2]="all,1,0";//Key
-$ThisPageSearchCriteriaDisplayRadioMaster[3]= array( xl("All"),xl("Coded"),xl("Not Coded") );//Display Value
-$ThisPageSearchCriteriaRadioKeyMaster[3]="all,not null,null";//Key
-$ThisPageSearchCriteriaDisplayRadioMaster[4]= array( xl("All"),xl("Unbilled"),xl("Billed"),xl("Denied") );//Display Value
-$ThisPageSearchCriteriaRadioKeyMaster[4]="all,0,1,7";//Key
-$ThisPageSearchCriteriaDisplayRadioMaster[5]= array( xl("All"),xl("Authorized"),xl("Unauthorized") );
-$ThisPageSearchCriteriaRadioKeyMaster[5]="%,1,0";
-$ThisPageSearchCriteriaDisplayRadioMaster[6]= array( xl("All"),xl("None"),xl("Ins 1"),xl("Ins 2 or Ins 3") );
-$ThisPageSearchCriteriaRadioKeyMaster[6]="all,0,1,2";
-//The below section is needed if there is any 'query_drop_down' type in the $ThisPageSearchCriteriaDataTypeMaster
-$ThisPageSearchCriteriaQueryDropDownMaster[1]="SELECT name,id FROM x12_partners;";
-$ThisPageSearchCriteriaQueryDropDownMasterDefault[1]= xl("All");//Only one item will be here
-$ThisPageSearchCriteriaQueryDropDownMasterDefaultKey[1]="all";//Only one item will be here
-//The below section is needed if there is any 'include' type in the $ThisPageSearchCriteriaDataTypeMaster
-//Function name is added here.Corresponding include files need to be included in the respective pages as done in this page.
-//It is labled(Included for Insurance ajax criteria)(Line:-279-299).
-$ThisPageSearchCriteriaIncludeMaster[1]="InsuranceCompanyDisplay";//This is php function defined in the file 'report.inc.php'
-
-if(!isset($_REQUEST['mode']))//default case
- {
-    $_REQUEST['final_this_page_criteria'][0]="(form_encounter.date between '".date("Y-m-d 00:00:00")."' and '".date("Y-m-d 23:59:59")."')";
-    $_REQUEST['final_this_page_criteria'][1]="billing.billed = '0'";
-
-    $_REQUEST['final_this_page_criteria_text'][0]=xl("Date of Service = Today");
-    $_REQUEST['final_this_page_criteria_text'][1]=xl("Billing Status = Unbilled");
-
-    $_REQUEST['date_master_criteria_form_encounter_date']="today";
-    $_REQUEST['master_from_date_form_encounter_date']=date("Y-m-d");
-    $_REQUEST['master_to_date_form_encounter_date']=date("Y-m-d");
-
-    $_REQUEST['radio_billing_billed']=0;
+    $_REQUEST['date_master_criteria_form_encounter_date'] = "today";
+    $_REQUEST['master_from_date_form_encounter_date'] = date("Y-m-d");
+    $_REQUEST['master_to_date_form_encounter_date'] = date("Y-m-d");
 
+    $_REQUEST['radio_billing_billed'] = 0;
 }
 ?>
-<table width='100%' border="0" cellspacing="0" cellpadding="0">
- <tr>
-      <td width="25%">&nbsp;</td>
-      <td width="50%">
+<table
```

Comment:
Drop the inline styling in favor of `table` class (Can also add `table-condensed table-striped table-hover`. In the near future I'll be adding a SCSS pre-compiler so `table` will include striped and hover classes by default.